### PR TITLE
Solar change deduplication model

### PR DESCRIPTION
**Pull request by zapililirad**
Solar Appscreener parcer:
Correct settings to apply hashcode as deduplication algoritm
Add line field to finding in parcer

### CODE REVIEW

File: dojo/settings/settings.dist.py [hunk 1]
```diff
@@ -1073,6 +1074,7 @@ def saml2_attrib_map_format(dict):
     'Hadolint Dockerfile check': True,
     'Semgrep JSON Report': True,
     'Generic Findings Import': True,
+    'Solar Appscreener Scan': True,
```
Comment (StefanFl) [inline dojo/settings/settings.dist.py:1077]: This line is not necessary, because CWE is not in the hash_code computation

**Review by StefanFl — CHANGES_REQUESTED** [review]
Thanks for this PR, one small change requested.

File: dojo/settings/settings.dist.py (hunk 1 above)
Comment (damiencarol) [reply dojo/settings/settings.dist.py:1077]: I agree. please remove this change only and we can move forward.

Comment (damiencarol) [inline dojo/settings/settings.dist.py:1077]:
```suggestion
```

**Comment by zapililirad** [comment]
done

**Review by StefanFl — APPROVED** [review]

**Pull request merged**